frontend/packages/pipelines-plugin/src/components/import/pipeline/PacInfoPanel.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { ClipboardCopy, ClipboardCopyVariant, Text } from '@patternfly/react-core';
+import { FormikValues, useFormikContext } from 'formik';
+import { Trans, useTranslation } from 'react-i18next';
+import {
+  ReadableResourcesNames,
+  Resources,
+} from '@console/dev-console/src/components/import/import-types';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+import { useActiveNamespace } from '@console/shared/src';
+import './PacSection.scss';
+
+const InfoPanel: React.FC = () => {
+  const { t } = useTranslation();
+  const [namespace] = useActiveNamespace();
+  const { values } = useFormikContext<FormikValues>();
+  const selectedResource = ReadableResourcesNames[values.resources as Resources];
+
+  return (
+    <FormSection>
+      <Text className="odc-pipeline-section-pac__info-panel">
+        <Trans t={t} ns="pipelines-plugin">
+          <div className="odc-pipeline-section-pac__info-text">
+            Once your Pipeline Repository is configured, in order to update your{' '}
+            {selectedResource && t(selectedResource.toLowerCase())}{' '}
+            <code className="co-code">{values.name}</code> automatically, update the following in
+            your <code className="co-code">.tekton</code> PipelineRun:
+          </div>
+          <ul>
+            <li>
+              <strong> {t(selectedResource)}: </strong>
+              <ClipboardCopy variant={ClipboardCopyVariant.inlineCompact}>
+                {values.name}
+              </ClipboardCopy>
+            </li>
+            <li className="odc-pipeline-section-pac__clipboard">
+              <strong> Internal image registry: </strong>
```

Comment:
Can we just translate this single string here to simplify the exported translation strings:

```suggestion
              <strong>{t('pipelines-plugin~Internal image registry:')}{' '}</strong>
```